docs/dam/integration_with_dxp.md

```diff
@@ -0,0 +1,180 @@
+---
+description: Integrate DAM with your Ibexa DXP project to use an effective tool for managing marketing assets.
+---
+
+
+# Integrate DAM with DXP
+
+DAM integration with [[= product_name =]] is dedicated to [[= product_name_base =]] users who also use Actito services. It enables using assets repository stored in the DXP in the Actito platform to build and insert marketing assets into campaign emails.
+For more information, see [Actito documentation](https://cdn3.actito.com/fe/actito-documentation/docs/Using_Assets_from_a_DAM/#connection-to-ibexa).
+
+## Enable Elasticsearch or Solr
+
+To ensure proper indexing and search results, DAM requires one of
+the following search engines on your server:
+
+- [Elasticsearch](install_elastic_search.md)
+- [Solr](install_solr.md)
+
+
+## Enable OAuth Servers
+
+Next, enable 0Auth Server authentication. To do it, follow steps up to the [Client section](oauth_server.md).
+
+## Install DAM package
+
+Install dedicated DAM `ibexa/dam-user` package into your [[= product_name =]] instance.
+
+To install, run the following command:
+
+```bash
+composer require ibexa/dam-user
+```
+
+Add the `ibexa_dam_oauth2_client_user` table to the database:
+
+```bash
+php bin/console ibexa:doctrine:schema:dump-sql vendor/ibexa/dam-user/src/bundle/Resources/config/schema.yaml | mysql -u <username> -p <database_name>
+```
+
+Next, add technical users, required by DAM to your [[= product_name =]] instance.
```

Comment:
It is worth to mention about two technical users that will be created

- `image-picker-reader` - with read permission 
- `image-picker-editor` - with edit / delete permission 

Both users have limited access to Media folder only.